sparkmagic/sparkmagic/tests/test_sqlquery.py

```diff
@@ -86,6 +86,11 @@ def test_pyspark_livy_sql_options():
                           .format(LONG_RANDOM_VARIABLE_NAME, query,
                                   LONG_RANDOM_VARIABLE_NAME, conf.pyspark_sql_encoding())))
 
+    sqlquery = SQLQuery(query, samplemethod='sample', samplefraction=0.33, maxrows=3234)
```

Comment:
Do you want to test that `to_command` with pyspark3 creates the right command too?